src/Uno.Templates/content/unoapp/MyExtensionsApp.1/App.recommended.cs

```diff
@@ -122,16 +122,23 @@ protected async override void OnLaunched(LaunchActivatedEventArgs args)
                 .UseSerialization((context, services) => services
                     .AddContentSerializer(context)
                     .AddJsonTypeInfo(WeatherForecastContext.Default.IImmutableListWeatherForecast))
-                .UseHttp((context, services) => services
-                    // Register HttpClient
-//-:cnd:noEmit
```

Comment:
I think we need these `noEmit` comments for something, I believe?

@erikvilima / @nickrandolph , correct?